server/main.lua

```diff
@@ -13,136 +14,107 @@ function ExploitBan(id, reason)
         'qb-management'
     })
 
-    TriggerEvent('qb-log:server:CreateLog', 'bans', "Player Banned", 'red', string.format("%s was banned by %s for %s", GetPlayerName(id), 'qb-management', reason), true)
+    TriggerEvent('qb-log:server:CreateLog', 'bans', 'Player Banned', 'red', string.format('%s was banned by %s for %s', GetPlayerName(id), 'qbx_management', reason), true)
 
-    DropPlayer(id, "You were permanently banned by the server for: Exploiting")
+    DropPlayer(id, 'You were permanently banned by the server for: Exploiting')
 end
 
+
 ---@param account string
 ---@return number
-function GetAccount(account)
+lib.callback.register('qbx_management:server:getAccount', function(_, account)
 	return exports['Renewed-Banking']:getAccountMoney(account) or 0
-end
-
-exports('GetAccount', GetAccount)
-exports('GetGangAccount', GetAccount)
+end)
 
 ---@param account string
 ---@param amount number
 function AddMoney(account, amount)
-	if not Accounts[account] then
-		Accounts[account] = 0
-	end
-
-	Accounts[account] += amount
-	exports['Renewed-Banking']:addAccountMoney(account, amount)
+	exports['Renewed-Banking']:addAccountMoney(account, amount, 'qbx_management')
 end
 
 ---@param account string
 ---@param amount number
 ---@return boolean
 function RemoveMoney(account, amount)
```

Comment:
```suggestion
local function RemoveMoney(account, amount)
```

Since we are going to 1 resource file we need all of these functions to be localized

Reply:
Changed